src/components/RunTestsButton.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { TOOL_ID } from "../constants";
+import { RunTestsButton } from "./RunTestsButton";
+
+const meta = {
+  component: RunTestsButton,
+  args: {
+    isStarting: false,
+    projectId: "projectId",
+    accessToken: "accessToken",
+    startBuild: action("startBuild"),
+    key: TOOL_ID,
```

Comment:
I don't think you are allowed to have a prop called `key` in React?

Reply:
I don't believe this would be necessary any more anyway, I think the `key` prop was just there to distinguish otherwise identical `IconButtons`

Reply:
Yes I meant to remove this